packages/template/vite-typescript/tmpl/forge.env.d.ts

```diff
@@ -1 +1,6 @@
 /// <reference types="@electron-forge/plugin-vite/forge-vite-env" />
+
+declare module 'electron-squirrel-startup' {
```

Comment:
Should we just use https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/electron-squirrel-startup instead of re-declaring the type here?

Reply:
Okay